src/AudioAnalysisTools/DSP/NoiseRemoval_Briggs.cs

```diff
@@ -125,20 +87,22 @@ public static class NoiseRemoval_Briggs
             return outM;
         }
 
+        // # NEXT FOUR METHODS ARE FOR LOCAL CONTRAST NORMALISATION ###############################################################################################
+
         /// <summary>
         /// Assumes the passed matrix is a spectrogram. i.e. rows=frames, cols=freq bins.
-        /// Does column-wise LCN (Local Contrast Normalisation.
-        /// The denominator = (contrastLevel + Math.Sqrt(localVariance[y])
+        /// First obtains background noise profile calculated from lowest 20% of cells for each freq bin independently.
+        /// Loop over freq bins (columns) - subtract noise and divide by LCN (Local Contrast Normalisation.
+        ///
+        /// The LCN denominator = (contrastLevelConstant + Sqrt(localVariance[y])
+        /// Note that sqrt of variance = std dev.
         /// A low contrastLevel = 0.1 give more grey image.
         /// A high contrastLevel = 1.0 give mostly white high contrast image.
-        /// It tried various other normalisation equations as can be seen below.
-        /// Taking square-root of top line results in too much background.
-        /// The algorithm is not overly sensitive to the neighbourhood size.
         /// </summary>
-        /// <param name="neighbourhood">suitable vaues are odd numbers 9 - 59.</param>
-        /// <param name="contrastLevel">Suitable values are 0.1 to 1.0.</param>
-        public static double[,] NoiseReduction_byLCNDivision(double[,] matrix, int neighbourhood, double contrastLevel)
+        public static double[,] NoiseReduction_byLCN(double[,] matrix, int lowPercent, int neighbourhood, double contrastLevel)
```

Comment:
Violates naming convention.
```suggestion
        public static double[,] NoiseReductionByLCN(double[,] matrix, int lowPercent, int neighbourhood, double contrastLevel)
```